### PR TITLE
Feature/communication values

### PR DESCRIPTION
**Pull request by kouroshD**
Reference Comment: 
- https://github.com/robotology/walking-teleoperation/pull/72#discussion_r781085256
- https://github.com/robotology/walking-teleoperation/pull/72#issuecomment-1015219491

### CODE REVIEW

File: modules/HapticGlove_module/src/GloveWearable.cpp [hunk 1]
```diff
@@ -354,7 +354,7 @@ bool GloveWearableImpl::setFingertipForceFeedbackValues(const std::vector<int>&
         wearableActuatorCommand.info.status = wearable::msg::ActuatorStatus::OK;
         wearableActuatorCommand.duration = 0;
 
-        m_iWearActuatorPort.write(false);
+        m_iWearActuatorPort.write(true);
```
Comment (S-Dafarra) [inline modules/HapticGlove_module/src/GloveWearable.cpp:357]: Can you add a line of documentation why this is needed? The same below

Comment (kouroshD) [reply modules/HapticGlove_module/src/GloveWearable.cpp]: Done.

File: modules/HapticGlove_module/src/GloveWearable.cpp [hunk 2]
```diff
@@ -385,7 +386,7 @@ bool GloveWearableImpl::setFingertipVibrotactileValues(const std::vector<int>& v
...
```
Comment (S-Dafarra) [inline modules/HapticGlove_module/src/GloveWearable.cpp:389]: Can you add the documentation line also here?

**Comment by kouroshD** [comment]
@S-Dafarra  is there any other comment, or we can continue with the merge after the CI checks?

**Review by S-Dafarra — APPROVED** [review]

**Comment by S-Dafarra** [comment]
> @S-Dafarra is there any other comment, or we can continue with the merge after the CI checks?

Looks good to me! Thanks

**Review by traversaro — APPROVED** [review]

**Comment by kouroshD** [comment]
thanks, merging.

**Pull request merged**